src/components/tabs/tab-item/RuiTabItem.vue

```diff
@@ -24,19 +22,24 @@ const css = useCssModule();
 
 <template>
   <div :class="[css.tab, { 'active-tab-item': active }]">
-    <TransitionRoot
+    <Transition
       :key="reverse.toString()"
```

Comment:
does the key here really needed? The `reverse` transition not working with this (tab 3 => tab 1)

Reply:
I guess it was only needed for TransitionRoot, maybe you can remove it too 😆